src/justenoughharfbuzz.c

```diff
@@ -254,6 +256,12 @@ int shape (lua_State *L) {
       lua_pushstring(L, "depth");
       lua_pushnumber(L, -tHeight - height);
       lua_settable(L, -3);
+      lua_pushstring(L, "x_bearing");
+      lua_pushnumber(L, x_bearing);
+      lua_settable(L, -3);
+      lua_pushstring(L, "glyphWidth");
+      lua_pushnumber(L, glyphWidth);
+      lua_settable(L, -3);
```

Comment:
You might as well push the whole glyph extents and be done with it. Other code might need axis to some of it.